prov/verbs/src/verbs_info.c

```diff
@@ -680,8 +681,9 @@ static int fi_ibv_alloc_info(struct ibv_context *ctx, struct fi_info **info,
 		goto err;
 	}
 
-	snprintf(fi->domain_attr->name, name_len + 1, "%s%s",
-		 ctx->device->name, ep_dom->suffix);
+	if (snprintf(fi->domain_attr->name, name_len + 1, "%s%s",
+		 ctx->device->name, ep_dom->suffix) >= name_len + 1)
+		 fi->domain_attr->name[name_len] = '\0';
```

Comment:
This is a duplicate of the next line

Reply:
Let me check that.  I was kind of rushing through this patch and may have missed this.

Reply:
Ok stupid me...

would you just prefer to always assign the last char to '\0' like this code rather than checking the return value?  I thought about doing that as it still protects us the same way.